homeassistant/components/media_player/services.yaml

```diff
@@ -323,7 +323,7 @@ squeezebox_call_method:
 
 yamaha_enable_output:
   description: Enable or disable an output port
-
+  
```

Comment:
Please remove this white space.

Reply:
Ok, I'll remove this, I did it first, but then I noticed that thats how it is in current dev branch and I didn't think it was for me to correct. But I will of course do it.